proto/tabletmanagerdata.proto

```diff
@@ -621,6 +621,7 @@ message VDiffCoreOptions {
   int64 timeout_seconds = 6;
   int64 max_extra_rows_to_compare = 7;
   bool update_table_stats = 8;
+  int64 max_diff_seconds = 9;
```

Comment:
this is more a general question, since the other duration-type in this message is also `int64`: is there a reason we do the "int as seconds" thing here rather than using `vttime.Duration` like we do in vtctldata and others?

(also extremely nit-picky and GH won't let me comment on the line, but i'm spotting a double-space:

```diff
- int64  max_rows = 3;
+ int64 max_rows = 3;
```

Reply:
I thought about that too, but vttime is a struct with a seconds and nanoseconds value and in this case I only cared about the seconds so just used that part of it. I'm fine changing it back though too.

I'll fix the spacing. Thanks!

Reply:
i'm good with the consistency! (i.e. don't change it)